README.md

```diff
@@ -40,21 +40,28 @@ Check out the [getting started guide](./docs/getting-started.md) for launching a
+| DigitalOcean Provider v1alpha4 (v0.5) |                             |                             |                             | ✓                           |                       |
+| DigitalOcean Provider v1       (v1.0) |                             |                             |                             | ✓                           |                       |
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-||Kubernetes 1.13|Kubernetes 1.14|Kubernetes 1.15|Kubernetes 1.16|Kubernetes 1.17|Kubernetes 1.18|Kubernetes 1.19|
-|-|-|-|-|-|-|-|-|
-|DigitalOcean Provider v1alpha1 (v0.1)|✓|✓|✓|||||
-|DigitalOcean Provider v1alpha1 (v0.2)|✓|✓|✓|||||
-|DigitalOcean Provider v1alpha2 (v0.3)||✓|✓|✓|✓|||
-|DigitalOcean Provider v1alpha3 (v0.4)||||✓|✓|✓|✓|
+|                 | DigitalOcean Provider v1alpha1 (v0.1) | DigitalOcean Provider v1alpha1 (v0.2) | DigitalOcean Provider v1alpha2 (v0.3) | DigitalOcean Provider v1alpha3 (v0.4) | DigitalOcean Provider v1alpha4 (v0.5) | DigitalOcean Provider v1 (v1.0) |
+| --------------- | ------------------------------------- | ------------------------------------- | ------------------------------------- | ------------------------------------- | ------------------------------------- | ------------------------------- |
+| Kubernetes 1.13 | ✓                                     | ✓                                     |                                       |                                       |                                       |                                 |
+| Kubernetes 1.14 | ✓                                     | ✓                                     |                                       |                                       |                                       |                                 |
+| Kubernetes 1.15 | ✓                                     | ✓                                     |                                       |                                       |                                       |                                 |
+| Kubernetes 1.16 |                                       |                                       | ✓                                     |                                       |                                       |                                 |
+| Kubernetes 1.17 |                                       |                                       | ✓                                     | ✓                                     |                                       |                                 |
+| Kubernetes 1.18 |                                       |                                       |                                       | ✓                                     |                                       | ✓(?)                            |
```

Comment:
The colums with a `(?)` need special review attention as I wasn't certain about our compatibility.